### DISCUSSION TITLE
Add bib pattern to site

### DISCUSSION THREAD

**Discussion by darigovresearch** [freesewing/freesewing#1622] [Forum]
**What is it that you would like to see happen?**

When looking to learn how to make a pattern for #1431 we found the tutorial for how to make a bib pattern on the documentation site - https://freesewing.dev/tutorials/pattern-design/

Would like to see the bib sewing pattern in the site of the designs (https://freesewing.org/designs/) as it is a useful item of clothing & may be a good introductory sewing pattern for a new user or workshop. Also when the item is made it could potentially be "upcycled" into another item of clothing so the materials would not be wasted

**Are you a FreeSewing patron?**

 - [ ] Yes, I am :hugs:
 - [x] No, I am not :thinking:

**Additional context**

Would be happy to make a pull request after replicating the bib after attempting the tutorial. Or if the code already exists somewhere to use that as the source to make a pull request. Is there an existing file that someone can point us to?

Also anybody who is new to contributing or interested in helping is welcome to join. See this video if you need some guidance on how you may be able to help - https://www.youtube.com/watch?v=UWA4wyacY2A

Suggested name is Bob the bib

**Comment by darigovresearch** [comment]
Bib code is currently already in the site as mentioned on the contributor call that this was raised on, it just needs to be ported with the relevant documentation pages to be visible on the design site - https://freesewing.org/designs/

https://github.com/freesewing/freesewing/blob/develop/packages/tutorial/src/bib.js

**Comment by bobgeorgethe3rd** [comment]
This has been completed with Pattern Bob

**Comment by darigovresearch** [comment]
Link is available here - https://freesewing.org/designs/bob/

**Comment by darigovresearch** [comment]
Link is available here - https://freesewing.org/designs/bob/

**Comment by joostdecock** [comment]
Done.

**Comment by darigovresearch** [comment]
Thanks!
